tests/ImportCommentsPreserved/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -0,0 +1,178 @@
+// This is part of multiple-lines at document-top
+// Second line followed by a gap
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// This is part of multiple-lines at document-top
+// Second line followed by a gap
+
+import a from "a";
+import b from "b";
+
+`;
+
+exports[`import-comments-document-top-test2.ts - typescript-verify > import-comments-document-top-test2.ts 1`] = `
+/**
+ * This is part of multiple-lines at document-top
+ * Second line of text, comment is followed by a gap (gap gets eaten)
+ */
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/**
+ * This is part of multiple-lines at document-top
+ * Second line of text, comment is followed by a gap (gap gets eaten)
+ */
+
+import a from "a";
+import b from "b";
+
+`;
+
+exports[`import-comments-document-top-test3.ts - typescript-verify > import-comments-document-top-test3.ts 1`] = `
+/**
+ * This comment is before the whole block,
+ * it is not attached to the following import.
+ */
+
+import b from "b";
+import a from "a";
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/**
+ * This comment is before the whole block,
+ * it is not attached to the following import.
+ */
+
+import a from "a";
+import b from "b";
+
+`;
+
+exports[`import-comments-outer-comments-block-comments.ts - typescript-verify > import-comments-outer-comments-block-comments.ts 1`] = `
+// Loose leading comment before imports (should not be dragged down with B)
+
+/* Leading comment before B */
+import B from "b"; /* Trailing comment on same-line as B (babel doesn't like CommentBlocks on same-line as anything else, so it will be shifted to a clean line) */
+/* Trailing comment on first line after B (This is treated as a leading comment for next import A!) */
+
+
+/* Leading comment before A */
+import A from "a"; /* Trailing comment on same-line as A (babel doesn't like CommentBlocks on same-line as anything else, so it will be shifted to a clean line) */
+/* Trailing comment on first line after A (will initially, because this is "close" to last-import 'a' */
+
+// Loose trailing comment after imports (should not be dragged up with A)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// Loose leading comment before imports (should not be dragged down with B)
+
+/* Trailing comment on same-line as B (babel doesn't like CommentBlocks on same-line as anything else, so it will be shifted to a clean line) */
+/* Trailing comment on first line after B (This is treated as a leading comment for next import A!) */
+
+/* Leading comment before A */
+import A from "a"; /* Trailing comment on same-line as A (babel doesn't like CommentBlocks on same-line as anything else, so it will be shifted to a clean line) */
```

Comment:
Oddly, this one is _not_ shifted to a new line.

Reply:
This comment was added before today’s changes, it no-longer gets force-wrapped.

I’ll fix the wording!